### PR TITLE
Tooltip Gallery demo

### LINKED ISSUE #1572: Gallery App has Tooltips
From https://www.google.com/design/spec/components/tooltips.html#

- [ ] press and hold trigger tooltip

### PR DESCRIPTION
**Pull request by HansMuller**
Also: added a size parameter to IconButton; just a pass-through to Icon.

Fixes https://github.com/flutter/flutter/issues/1572

### CODE REVIEW

**Comment by abarth** [comment]
LGTM

File: examples/material_gallery/lib/demo/tooltip_demo.dart [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      ),
+      body: new Builder(
+        builder: (BuildContext context) {
+          return new Column(
+            alignItems: FlexAlignItems.stretch,
+            children: <Widget>[
+              new Text(_introText, style: theme.text.subhead),
+              new Row(
+                children: <Widget>[
+                  new Text('Long press the ', style: theme.text.subhead),
```
Comment (Hixie) [inline examples/material_gallery/lib/demo/tooltip_demo.dart]: @abarth this is a use case for inline images, btw

Comment (abarth) [inline examples/material_gallery/lib/demo/tooltip_demo.dart]: Yeah.

Comment (Hixie) [inline examples/material_gallery/lib/demo/tooltip_demo.dart]: Filed https://github.com/flutter/flutter/issues/2022 for it.

**Comment by Hixie** [comment]
Might be worth having an example in there that uses a Tooltip directly.

**Comment by HansMuller** [comment]
I've added a Tooltip around the inline call icon.

**Pull request merged**